### PR TITLE
ISSUE-TEMPLATES: add Let's document; update others

### PR DESCRIPTION
**Pull request by acuteenvy**
### Changes to existing templates
- Set some fields as required
- Change OS selection from checkboxes to dropdown, because we only need to know the directory name anyway
- Add a field for documentation
- Remove most fields from the translation template, because if a page already exists we only need to know its name

#### NOTE: merge this after #11313 because this adds BSDs to the platform dropdown.

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]
Looks good to me

**Comment by kbdharun** [comment]
## Changes I made

- Updated the Let's Document label from "Let's document" to "let's document" (to align with the other non-capitalized labels).
- Capitalized issue titles.
- Changed OS -> Platform and alphabetically arranged them.
- Added additional information tab to document information when a command i.e. curl differs between platforms.
- Add help wanted label to the translation request template.

Feel free to test out the changes here https://github.com/kbdharun/tldr/issues/new/choose will have issue tab open until this PR is merged.

**Review by kbdharun — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/lets-document.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        - SunOS
+        - Windows
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: Provide additional information if the command differs between platforms.
+    validations:
+      required: false
```
Comment (acuteenvy) [inline .github/ISSUE_TEMPLATE/lets-document.yml:39]: `required: false` is the default.
```suggestion
```

File: .github/ISSUE_TEMPLATE/page-request.yml [hunk 2]
```diff
@@ -1,26 +1,44 @@
...
+        - SunOS
+        - Windows
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: Provide additional information if the command differs between platforms.
+    validations:
+      required: false
```
Comment (acuteenvy) [inline .github/ISSUE_TEMPLATE/page-request.yml:44]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/page-modification-request.yml [hunk 3]
```diff
@@ -1,25 +1,44 @@
...
+        - SunOS
+        - Windows
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional information
+      description: Provide additional information if the command differs between platforms.
+    validations:
+      required: false
```
Comment (acuteenvy) [inline .github/ISSUE_TEMPLATE/page-modification-request.yml:44]:
```suggestion
```

**Review by acuteenvy — COMMENTED** [review]
"Additional information" and "Command details" are pretty similar. Maybe we could merge them into one field?

**Comment by kbdharun** [comment]
> "Additional information" and "Command details" are pretty similar. Maybe we could merge them into one field?

Kinda agree, but I think we can indeed keep it as it is. Let's see what others think.

**Pull request merged**